config/platforms/openshift/che-operator.clusterserviceversion-description-patch.yaml

```diff
@@ -0,0 +1,66 @@
+#
+#  Copyright (c) 2012-2021 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+
+- op: replace
+  path: /spec/description
+  value: |
+    A collaborative Kubernetes-native development solution that delivers OpenShift workspaces and in-browser IDE for rapid cloud application development.
+    This operator installs PostgreSQL, Keycloak, and the Eclipse Che server, as well as configures all three services.
+
+    ## How to Install
+    Press the **Install** button, choose the channel and the upgrade strategy, and wait for the **Installed** Operator status.
+    When the operator is installed, create a new CR of Kind CheCluster (click the **Create New** button).
+    The CR spec contains all defaults (see below).
+    You can start using Eclipse Che when the CR status is set to **Available**, and you see a URL to Eclipse Che.
+
+    ## Defaults
+    By default, the operator deploys Eclipse Che with:
+    * Bundled PostgreSQL and Keycloak
+    * Common PVC strategy
+    * Auto-generated passwords
+    * TLS mode (secure routes)
+    * Communicate between components using internal cluster SVC names
+    * Regular login extended with OpenShift OAuth authentication
+
+    ## Installation Options
+    Eclipse Che operator installation options include:
+    * Connection to external database and Keycloak
+    * Configuration of default passwords and object names
+    * PVC strategy (once shared PVC for all workspaces, PVC per workspace, or PVC per volume)
+    * Authentication options
+
+    ### External Database and Keycloak
```

Comment:
Can we keep it clean and just add links to docs?

Reply:
We don't have such a doc

Reply:
Do we really miss a doc on how to configure external Keycloak or Postgres?

Reply:
We have a Postgres doc but not keycloak

Reply:
found 
https://www.eclipse.org/che/docs/che-7/administration-guide/configuring-authorization/#configuring-che-to-use-external-keycloak_che